core/src/test/java/de/danoeh/antennapod/core/storage/DbWriterTest.java

```diff
@@ -599,18 +598,20 @@ public void testAddQueueItemSingleItemAlreadyInQueue() throws InterruptedExcepti
     }
 
     @Test
-    public void testAddQueueItemMultipleItems() throws InterruptedException, ExecutionException, TimeoutException {
-        final int NUM_ITEMS = 10;
+    public void testAddQueueItemMultipleItems() throws Exception {
+        final int numItems = 10;
 
-        Feed feed = queueTestSetupMultipleItems(NUM_ITEMS);
+        Feed feed;
+        feed = queueTestSetupMultipleItems(numItems);
         PodDBAdapter adapter = PodDBAdapter.getInstance();
         adapter.open();
         Cursor cursor = adapter.getQueueIDCursor();
         assertTrue(cursor.moveToFirst());
-        assertEquals(NUM_ITEMS, cursor.getCount());
-        List<Long> expectedIds = FeedItemUtil.getIdList(feed.getItems());
+        assertEquals(numItems, cursor.getCount());
+        List<Long> expectedIds;
+        expectedIds = FeedItemUtil.getIdList(feed.getItems());
```

Comment:
Why did you separate that line?

Reply:
That's a workaround to satisfy the Checkstyle rule `VariableDeclarationUsageDistance` in `checkstyle-new-code.xml`. :grinning:
I couldn't just rearrange the code lines as the database requests must keep their order.

Reply:
Oh, right :D I think this class should also call `DbWriter.tearDownTests` to make sure that we don't introduce the CI issues again.

Reply:
Yes, you are right. I updated the code.